homeassistant/components/mqtt/mixins.py

```diff
@@ -243,7 +243,7 @@ def validator(config: ConfigType) -> ConfigType:
         if domain in warned:
             return config
 
-        _LOGGER.warning(
+        _LOGGER.error(
```

Comment:
Add a comment in this function about when it can be removed. Maybe removal can be done in 2023.6?